src/sound/sound-functions.js

```diff
@@ -0,0 +1,36 @@
+import { Howler, Howl } from 'howler';
+import { note } from 'tonal';
+
+
+export const sound = new Howl ({
+  src: ['../sound/pianosprite.mp3'],
```

Comment:
This is your main problem - React want's files like this to be imported and then used as a variable. It should fix the playback errors!

Reply:
Nice! I updated it. Thanks for leading me to this!